### PR TITLE
Initial import of prescriptions

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

Initial repository initialization.

Changed LICENSE to ODbL.

### CODE REVIEW

File: .aicoe-ci.yaml [hunk 1]
```diff
@@ -1,15 +1,2 @@
...
-# Uncomment following lines to build a public image of this repo
-# build:
-#   build-stratergy: Source
-#   build-source-script: "image:///opt/app-root/builder"
-#   base-image: quay.io/thoth-station/s2i-custom-notebook:latest
-#   registry: quay.io
-#   registry-org: aicoe
-#   registry-project: <CHANGE-ME>
-#   registry-secret: aicoe-pusher-secret
+check:
```
Comment (pacospace) [inline .aicoe-ci.yaml:1]: don't we use prow for pre-commit now?

Comment (fridex) [reply .aicoe-ci.yaml]: Adjusted :+1:

**Comment by fridex** [comment]
Will merge this for repo setup and open PRs for relevant entries separately.

**Review by pacospace — APPROVED** [review]
lgtm! 💯

**Pull request merged**